client.go

```diff
@@ -265,6 +266,12 @@ func setupChannel(remotePort int, sshPort int, login string) int {
 	return localPort
 }
 
+func (c *SingleServerClient) CleanupDeploy(deployIdForCleanup string) error {
```

Comment:
Seems your topic branching is wrong :) Probably want a new branch with just these changes